pageserver/src/tenant/metadata.rs

```diff
@@ -565,7 +565,7 @@ mod tests {
         );
         let expected_bytes = vec![
             /* TimelineMetadataHeader */
-            4, 37, 101, 34, 0, 70, 0, 4, // checksum, size, format_version (4 + 2 + 2)
```

Comment:
Curious, where does this come from? (No action required)

Reply:
The first 4 bytes are a checksum; since the Postgres version has changed, the checksum is also changed (I didn't calculate it, just took it from the error message 🙃 )